src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/Worker.java

```diff
@@ -0,0 +1,15 @@
+package edu.kit.ipd.crowdcontrol.objectservice.crowdworking;
+
+import java.util.Map;
+
+/**
+ * Created by marcel on 19.01.16.
+ */
+public interface Worker {
```

Comment:
i don`t think the name is optimal, i would name the interface IdentifyWorker and also mark it with @FunctionalInterface.